magefile.common.go

```diff
@@ -32,3 +35,53 @@ func downloadFile(url string, dest string) error {
 
 	return nil
 }
+
+func copyFile(destPath, srcPath string) error {
```

Comment:
I hope you don't really need because you have https://github.com/otiai10/copy/blob/master/copy.go

Reply:
Yes, you are right.